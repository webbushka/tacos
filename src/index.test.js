import Root from './';

describe('Component: index.js (Root)', () => {
  it('should not blow up', () => {
    const str = JSON.stringify(Object.assign({}, Root, { _reactInternalFiber: 'censored' }));
    expect(str).toMatchSnapshot();
  });
});
