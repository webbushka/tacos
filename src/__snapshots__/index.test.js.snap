// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: index.js (Root) should not blow up 1`] = `"{\\"props\\":{},\\"context\\":{},\\"refs\\":{},\\"updater\\":{},\\"state\\":{\\"tacos\\":[{\\"name\\":\\"Asada\\",\\"src\\":\\"asada.png\\"},{\\"name\\":\\"Carnitas\\",\\"src\\":\\"carnitas.png\\"},{\\"name\\":\\"Pollo\\",\\"src\\":\\"pollo.png\\"}]},\\"_reactInternalFiber\\":\\"censored\\",\\"_reactInternalInstance\\":{}}"`;
